src/components/Home.js

```diff
@@ -0,0 +1,177 @@
+import * as React from 'react';
+import { useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { styled } from '@mui/material/styles';
+import Grid from '@mui/material/Grid';
+import Paper from '@mui/material/Paper';
+import Box from '@mui/material/Box';
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
+import africaMap from '../assets/africa.png';
+import europeMap from '../assets/europe.png';
+import asiaMap from '../assets/asia.png';
+import australiaMap from '../assets/australia.png';
+import americaMap from '../assets/united-states-of-america.png';
+import Navbar from './Navbar';
+
```

Comment:
- [ ] As [requested](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md) for this project, " `When the page loads, the webapp shows the list of categories that could be filtered out by some parameter, for example by the category name.` ", please kindly add a component(`e.g search input`) that will enable a list of categories to be filtered using a given parameter(s), e.g. category name. This [article](https://www.freecodecamp.org/news/how-to-make-a-filter-component-in-react/) will throw more light on that.